includes/definitions/webpower-smart2.yaml

```diff
@@ -0,0 +1,15 @@
+os: webpower-smart2
```

Comment:
Just checking that this isn't better merged in with the current webpower os we have?

Reply:
> Just checking that this isn't better merged in with the current webpower os we have?

no, i tried to merge 2 OS in one yaml, but ...
old webpower is `rfc1628_compat`
and webpower2 is only partly compatible
so, if i merge them, then it will be a problem with messed up sensors discovery
it is best way to separate them